frontend/src/pages/UserInformations/UserInformations.jsx

```diff
@@ -0,0 +1,58 @@
+import BackButton from "../../components/BackButton/BackButton";
+import "./UserInformations.scss";
+import AddressIcon from "../../assets/address-21a89a.png";
+import GenderIcon from "../../assets/gender-mark-2-21a89a.png";
+import EmailIcon from "../../assets/email-21a89a.png";
+import BirthdayIcon from "../../assets/birthday-cake-21a89a.png";
+import userPicturePath from "../../assets/louiseampere.jpg";
+import arrowDark from "../../assets/arrowBackDark.svg";
+import NavigationButton from "../Home/NavigationButton";
+
+const users = {
+  id: 1,
+  firstname: "Louise",
+  lastname: "Ampère",
+  birthdate: "12 janvier 1980",
+  gender: "Femme",
+  email: "louise.ampere@tesla.com",
+  zipcode: "59000",
+  city: "Lille",
+};
+
+function UserInformations() {
+  return (
+    <section className="viewport_userinformations">
+      <img className="userpicture" alt="utilisateur" src={userPicturePath} />
```

Comment:
Non-informative images don't need an explanation in the alt attribute

Reply:
noted and removed